nexusutils/dataconverter/readers/em_spctrscpy/utils/em_utils.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
```

Comment:
is not it empty?

Reply:
It's empty but should be kept to guide to the place where parser-specific utils should be stored in the future

Reply:
But yeah dont mind can also remove it